Src/RE2.Managed/StringUtf8.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.InteropServices;
+
+namespace Microsoft.RE2.Managed
+{
+    [StructLayout(LayoutKind.Sequential, Pack = 1)]
+    public unsafe struct StringUtf8
```

Comment:
Similar to the existing `String8`.

Reply:
Moved to backlog: [https://github.com/microsoft/sarif-pattern-matcher/issues/386](https://github.com/microsoft/sarif-pattern-matcher/issues/386)